example/lib/main.dart

```diff
@@ -52,9 +49,47 @@ class _MyAppState extends State<MyApp> {
       setState(() {});
     };
 
+    FlutterAccessoryManager.onConnectionStateChanged =
+        (String deviceId, bool connected) {
+      print("Connection State Changed $deviceId $connected");
+      showSnackbar("$deviceId : ${connected ? 'Connected' : 'Disconnected'}");
+    };
+
+    FlutterAccessoryManager.onGetReport =
+        (String deviceId, reportType, reportId) {
+      print("Get Report $deviceId $reportType $reportId");
+
+      if (reportType != ReportType.input) {
+        return null; // Reply with error
+      }
+
+      return ReportReply(
+        data: null, // Reply with data
+      );
+    };
+
+    FlutterAccessoryManager.onSdpServiceRegistrationUpdate = (bool registered) {
+      print("Sdp Service Registered $registered");
+      showSnackbar("Sdp Service Registered $registered");
```

Comment:
```suggestion
      print("SDP Service Registered $registered");
      showSnackbar("SDP Service Registered $registered");
```
Let's use proper case in printable names and button labels